robot1:
  controller_manager:
    ros__parameters:
      update_rate: 100
      use_sim_time: true
      joint_state_broadcaster:
        type: joint_state_broadcaster/JointStateBroadcaster
      simple_diff_drive_controller:
        type: simple_diff_drive_controller/SimpleDiffDriveController

  simple_diff_drive_controller:
    ros__parameters:

      front_left_wheel_name: "front_left_wheel_joint"
      front_right_wheel_name: "front_right_wheel_joint"
      rear_left_wheel_name: "rear_left_wheel_joint"
      rear_right_wheel_name: "rear_right_wheel_joint"
      wheel_separation: 0.4
      wheelbase: 0.5
      wheel_radius: 0.15
      max_velocity: 10.0